app/main/views/briefs.py

```diff
@@ -130,9 +133,36 @@ def create_new_brief_response(brief_id):
             **dict(main.config['BASE_TEMPLATE_DATA'])
         ), 400
 
-    flash('Your response to &lsquo;{}&rsquo; has been submitted.'.format(brief['title']))
+    if all(brief_response['essentialRequirements']):
+        flash('Your response to ‘{}’ has been submitted.'.format(brief['title']))
+        return redirect(url_for(".dashboard"))
+    else:
+        return redirect(url_for(".view_response_result", brief_id=brief_id))
+
+
+@main.route('/opportunities/<int:brief_id>/responses/result')
+def view_response_result(brief_id):
+    brief = get_brief(data_api_client, brief_id, allowed_statuses=['live'])
+
+    if not is_supplier_eligible_for_brief(data_api_client, current_user.supplier_id, brief):
+        return _render_not_eligible_for_brief_error_page(brief)
 
-    return redirect(url_for(".dashboard"))
+    brief_response = data_api_client.find_brief_responses(
+        brief_id=brief_id,
+        supplier_id=current_user.supplier_id
+    )['briefResponses']
+
+    if len(brief_response) == 0:
+        result_state = 'not_submitted'
```

Comment:
Why not just redirect the user to the "respond to brief" page? No one without a response should visit this page during a normal flow anyway.